cms/static/js/spec/views/video_transcripts_spec.js

```diff
@@ -41,8 +46,39 @@ define(
                 );
 
                 expect(uploadTranscriptActionEl.html().trim(), 'Upload');
-                expect(uploadTranscriptActionEl.data('edx-video-id'), edxVideoID);
-                expect(uploadTranscriptActionEl.data('language-code'), transcriptLanguage);
+            };
+
+            verifyMessage = function($transcriptEl, status) {
+                var $transcriptStatusEl = $transcriptEl.find('.transcript-upload-status-container'),
+                    statusData = videoTranscriptsView.transcriptUploadStatuses[status];
+
+                expect($transcriptStatusEl.hasClass(statusData.statusClass)).toEqual(true);
+                expect($transcriptStatusEl.find('span.fa').hasClass(statusData.iconClasses)).toEqual(true);
+                expect(
+                    $transcriptStatusEl.find('.more-details-action').hasClass('hidden')
```

Comment:
we can assert the truth: `$transcriptStatusEl.find('.more-details-action').hasClass(statusData.hiddenClass)`

Reply:
Yes that was the plan but `statusData.hiddenClass` would be  `''` in case if there is an error so  `$transcriptStatusEl.find('.more-details-action').hasClass(statusData.hiddenClass)` would not fulfil our need.

Reply:
got it.